x-pack/plugins/cross_cluster_replication/public/app/components/follower_index_unfollow_provider.js

```diff
@@ -100,9 +100,9 @@ class Provider extends PureComponent {
               <p>
                 <FormattedMessage
                   id="xpack.crossClusterReplication.unfollowLeaderIndex.confirmModal.multipleUnfollowDescription"
-                  defaultMessage="The follower indices below will be paused, closed, and converted into regular indices.
-                    The indices will no longer appear in Cross Cluster Replication but can still be managed using Index
-                    Management. This operation cannot be undone."
+                  defaultMessage="These follower indices will be converted to standard indices and no
+                    longer appear in Cross Cluster Replication, but you can manage them in Index
```

Comment:
The follower indices will be converted to standard indices.  They will no longer appear ....